packages/frontend/web/src/index.tsx

```diff
@@ -2,6 +2,7 @@ import './polyfill/dispose';
 import './polyfill/intl-segmenter';
 import './polyfill/promise-with-resolvers';
 import './polyfill/request-idle-callback';
+import './polyfill/set-immediate';
```

Comment:
should we add this to electron/renderer/index.tsx ?

Reply:
I will remove it for now. It seems only shell.tsx need it.
We can revisit the polyfill later